crates/block-producer/src/server/mod.rs

```diff
@@ -87,7 +87,7 @@ impl BlockProducer {
         info!(target: COMPONENT, endpoint=?self.block_producer_address, store=%self.store_url, "Initializing server");
         let store = StoreClient::new(self.store_url.clone());
 
-        // Retry fetching the chain tip from the store until it succeeds.
+        // retry fetching the chain tip from the store until it succeeds.
```

Comment:
Nit: bit of a random change, could probably remain upper case

Reply:
Yeah, not even sure why/how I did this